client/packages/common/src/ui/layout/tables/components/Cells/CurrencyCell.tsx

```diff
@@ -28,12 +28,12 @@ export const CurrencyCell = <T extends RecordWithId>({
 > => {
   const { c } = useCurrency(currencyCode);
   const price = Number(column.accessor({ rowData }) ?? 0);
-  // format prices > 1 with default precision
-  const precision = price < 1 ? 10 : undefined;
-  const fullText = c(price, precision).format();
+  const fullText = c(price, 10).format();
```

Comment:
Just defaulting it to 10 dp for now... since we don't do precision customisations >.>